.github/build-elasticsearch-image.sh

```diff
@@ -0,0 +1,7 @@
+docker image build \
+    --tag ${DOCKER_USERNAME}/elasticsearch:${DOCKER_ELASTICSEARCH_IMAGE_TAG} \
+    --no-cache \
+    --compress \
+    -f project-base/docker/elasticsearch/Dockerfile \
+    . &&
+docker image push ${DOCKER_USERNAME}/elasticsearch:${DOCKER_ELASTICSEARCH_IMAGE_TAG}
```

Comment:
nitpick:
The tag could be extracted into variable as it is used on line 2 in a same way